docs/openapi/components/schemas/common/CTPATCertificate.yml

```diff
@@ -16,31 +16,111 @@ description: >-
+      - 'https://w3id.org/traceability/v1'
   type:
-    oneOf:
-      - type: array
-      - type: string
-        enum:
-          - CTPATCertificate
-  ctpatCertifiedEntity:
-    title: CTPAT Certified Entity
-    description: >-
-      In recognition of your commitment to partnerhip, and in appreciation for
-      joining with us to secure the international supply chain and protect our
-      country's security, The U.S. Customs Service is pleased to certify your
-      membership in the Customs - Trade Partnership Against Terrorism and to
-      present this certificate to
+    type: array
+    const:
+      - VerifiableCredential
+      - CTPATCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    $ref: ./Organization.yml
+  credentialSubject:
     $ref: ./Organization.yml
+  proof:
+    type: object
+  sviNumber:
+    title: SVI Number
+    description: >-
+      Status Verification Interface number.
+    type: string
+    $linkedData:
+      term: sviNumber
+      '@id': https://w3id.org/traceability#sviNumber
+  ctpatAccountNumber:
+    title: CTPAT Account Number
+    description: >-
+      CTPAT issued account number for certified entity.
+    type: string
+    $linkedData:
+      term: ctpatAccountNumber
+      '@id': https://w3id.org/traceability#ctpatAccountNumber  
+  tradeSector:
+    title: CTPAT Trade Sector
+    description: >-
+      Trade sectors elegible for CTPAT certification. 
+    enum:
+      - U.S. Importers
+      - Highway Carriers
+      - Foreign Manufacturers
+      - Consolidators
+      - Licensed U.S. Customs Brokers
+      - Mexican Long Haul Highway Carriers
+      - U.S. Exporters
+      - Third Party Logistics Providers
+      - Sea Carrier
+      - Marine Port Authority and Terminal Operators
+      - Foreign Based Marine Port Terminal Operators
+      - Air Carriers
+      - Rail Carriers
     $linkedData:
-      term: ctpatCertifiedEntity
-      '@id': https://w3id.org/traceability#ctpatCertifiedEntity
-additionalProperties: false
-required:
-  - ctpatCertifiedEntity
+      term: tradeSector
+      '@id': https://schema.org/industry
+  dateOfLastValidation:
+    title: Date of Last Validation
+    description: >-
+      Date of last validation.
+    type: string
+    $linkedData:
+      term: dateOfLastValidation
+      '@id': https://schema.org/Date
+  issuingCountry:
+    title: Issuing Country
+    description: >-
+      Issuing country.
+    type: string
+    $linkedData:
+      term: issuingCountry
+      '@id': https://schema.org/addressCountry
+additionalProperties: true
+required: []
 example: |-
   {
-    "type": "CTPATCertificate",
-    "ctpatCertifiedEntity": {
+    "type": [
+      "VerifiableCredential",
+      "CTPATCertificate"
+    ],
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      {
+        "@vocab": "https://w3id.org/traceability/v1/vocab#"
```

Comment:
does not appear to match the schema above, but we know we will fix this, later.

Reply:
No, this is to solve the chicken-egg situation involved with introducing new terms now. It has to be a two-PR process (1. define the new terms, once published 2. use those new terms) - quite cumbersome. A tool for short circuiting this process seems essential for productivity.